tests/privacy.test.js

```diff
@@ -0,0 +1,43 @@
+const { runWPTTest } = require('./wpt.js');
+const path = require('path');
+const options = {
+  label: path.basename(__filename).replace('.test.js', '')
+}
+
+beforeAll(async () => {
+  theverge = await runWPTTest("https://www.theverge.com/", options);
+  rtl = await runWPTTest("https://www.rtl.de/", options);
+  educations = await runWPTTest("https://www.educations.com/", options);
+  pokellector = await runWPTTest("https://www.pokellector.com/", options);
```

Comment:
Having so many external dependencies makes this test more likely to fail if any of them change. I wonder if we should have more contrived test cases on pages that we control to prevent that?

Also curious to get @tunetheweb's opinion.

Reply:
I've tried to do testing previously with sites like:
- https://global-privacy-control.glitch.me/
- https://user-agent-client-hints.glitch.me/
but it doesn't cover all the cases even for privacy chapter.

That would be awesome to have some more extended library of browser API and other demos.
Then we would be able to improve testing speed and maybe run all the test-cases with each action run.

Reply:
@tunetheweb any thoughts?